src/fiat-connect-client.ts

```diff
@@ -498,6 +517,10 @@ export class FiatConnectClientImpl implements FiatConnectApiClient {
       return handleError(error)
     }
   }
+
+  async getCookies(): Promise<String> {
```

Comment:
Is the output of [getCookieString](https://github.com/salesforce/tough-cookie#getcookiestring) as easily deserializable as the output of [serialize](https://github.com/salesforce/tough-cookie#serializecallbackerr-serializedobject)?

Reply:
getCookieString can be deserialized with Cookie.parse, while serialize goes with deserialize. 

getCookieString returns 'key=value'
serialize returns a custom struct

What do you recommend?